contracts/ligo/src/fixed_price_sale_market.mligo

```diff
@@ -89,16 +89,14 @@ let buy_token(sale, storage: sale_param * storage) : (operation list * storage)
   in
   let tx_money_op = transfer_money(sale.tokens.money_token_address, sale.tokens.money_token_token_id, sale.price, Tezos.sender, seller) in
   let tx_nft_op = transfer_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
-  let tx_nft_remove_operator_op = remove_operator_nft(sale.tokens.token_for_sale_address, sale.tokens.token_for_sale_token_id, Tezos.self_address, Tezos.sender) in
```

Comment:
consider removing dead code: `remove_operator_nft` and `add_operator_nft` functions